dataprep/eda/diff/render.py

```diff
@@ -0,0 +1,505 @@
+"""
+This module implements the visualization for the plot_diff function.
+"""  # pylint: disable=too-many-lines
+from typing import Any, Dict, List, Optional, Tuple
+
+import numpy as np
+import pandas as pd
+from bokeh.models import (
+    FuncTickFormatter,
+    HoverTool,
+)
+from bokeh.plotting import Figure
+from bokeh.transform import dodge
+
+from ..configs import Config
+from ..dtypes import Continuous, DateTime, Nominal, is_dtype
+from ..intermediate import Intermediate
+from ..palette import CATEGORY10
+
+__all__ = ["render_diff"]
+
+
+def tweak_figure(
```

Comment:
Are these functions (`tweak_figure()`, `format_ticks()`, ...) the same as from distribution/render.py? It might be better to move them into eda/utils.py if they are used in multiple places.